packages/patternfly-4/react-integration/demo-app-ts/src/components/demos/DonutUtilizationChartDemo/DonutUtilizationSimpleLeftDemo.tsx

```diff
@@ -0,0 +1,2 @@
+import React from 'react';
+import { ChartDonutUtilization } from '@patternfly/react-charts';
```

Comment:
Looks like we're missing an example for the donut utilization chart with thresholds?

I see 12 empty files

Reply:
Fixed.